code-challenges.js

```diff
@@ -21,49 +21,109 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
-
+// Create a function called boilingPoint that takes in a number
+//if number is less than 212 return below boiling point
+//else if number is euall to 212 return at boiling point 
+//else meaning any other number return above boiling point
```

Comment:
Good logic, but pay attention to the expected output. The prompt wants the argument as part of the answer.